src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,26 @@
+package qna.domain;
+
+import java.util.*;
+import java.util.stream.*;
+
+import javax.persistence.*;
+
+@Embeddable
+public class Answers {
+    @OneToMany(mappedBy = "question")
+    private final List<Answer> answers = new ArrayList<>();
+
+    protected Answers() {
+    }
+
+    public List<DeleteHistory> delete(User loginUser) {
```

Comment:
해당 메서드에 대한 변경이 이루어지지 않아 컴파일에러가 발생하고 있네요 ! 
확인 부탁드립니다 :)

추가적으로 DeleteHistory의 일급 컬렉션을 만들어주셨는데 DeleteHistories를 반환할때 사용해보는건 어떨까요?
```suggestion
    public DeleteHistories delete(User loginUser) {
```

Reply:
앗,  이상합니다.
분명히 List<DeleteHistory>에서 DeleteHistories로 변경하고 커밋하고 푸시까지 했는데요.
왜 리뷰어님에게는 과거의 소스로 보여질까요?